src/traces/sunburst/calc.js

```diff
@@ -192,7 +192,9 @@ exports.calc = function(gd, trace) {
     if(failed) return;
 
     // TODO add way to sort by height also?
-    hierarchy.sort(function(a, b) { return b.value - a.value; });
+    if ( fullLayout[ 'sort']) {
```

Comment:
After applying my comment https://github.com/plotly/plotly.js/pull/5164/files#r494581357, this line would become
```js
if(trace.sort) {
```